src/components/GeneralButton/GeneralButton.elements.js

```diff
@@ -0,0 +1,17 @@
+import styled from 'styled-components'
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/park-inn-ispp/park-inn-frontend/pullRequest?prid=9172775)
```suggestion
import styled from "styled-components"
```